spec/spec_helper.rb

```diff
@@ -1,2 +1,8 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'trix'
+require 'rails/version'
+require 'action_dispatch'
+require 'action_controller/metal/live'
+require 'action_controller/test_case'
```

Comment:
No, rspec/rails expects to be run within a rails project, where rails is loaded elsewhere. I only added these as needed to get things working.

> On Feb 3, 2016, at 11:41, Jon Moss <notifications@github.com> wrote:
> 
> In spec/spec_helper.rb:
> 
> > @@ -1,2 +1,8 @@
> >  $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
> >  require 'trix'
> > +require 'rails/version'
> > +require 'action_dispatch'
> > +require 'action_controller/metal/live'
> > +require 'action_controller/test_case'
> Wouldn't just requiring rspec/rails make all of these other requires not needed?
> 
> —
> Reply to this email directly or view it on GitHub.
>